ansible_rulebook/job_template_runner.py

```diff
@@ -61,6 +68,15 @@ async def _get_page(
     def _auth_headers(self) -> dict:
         return dict(Authorization=f"Bearer {self.token}")
 
+    @cached_property
+    def _sslcontext(self) -> Union[bool, ssl.SSLContext]:
+        if self.host.startswith("https"):
+            if self.verify_ssl.lower() == "yes":
+                return True
+            elif not self.verify_ssl.lower() == "no":
+                return ssl.create_default_context(cafile=self.verify_ssl)
```

Comment:
@bzwei should we check if the cafile is really a file and throw an error if its missing

Reply:
the method `create_default_context` will validate the file and throw an error.